### ISSUE TITLE
Recherche avancée : aide à la saisie

### ISSUE THREAD

**Issue by c-geek** [duniter/cesium#585]
Cesium permet d'effectuer des recherches avancées dans la blockchain, à l'aide de l'extension Cesium+.

Néanmoins, il n'y a pas d'aide quant à la syntaxe de recherche ni contrôle sur la saisie. Du coup on ne sait pas quoi taper et l'on fait des erreurs, qui comme ici mènent à une boucle d'erreur infinie : 

![image](https://user-images.githubusercontent.com/969136/31666352-2f20fb62-b34c-11e7-84b1-dcedcdd24a43.png)

**Comment by c-geek** [comment]
Il pourrait d'ailleurs être proposé quelques recherches par défaut : 

* tous les blocs avec transaction
* tous ceux avec nouveau venu
* ...

**Comment by blavenie** [comment]
Oui, j'avais en tete un assistant pour créer les requetes, par exemple un menu déroulant avec différentes mini-requete. Cela s'ajouterait à la requete courant, par exemple avec un `AND` par défaut. 

En attendant, je te propose l'aide en ligne ES : 
https://www.elastic.co/guide/en/elasticsearch/reference/2.4/query-dsl-query-string-query.html#query-string-syntax

Le nombre des attributes sont tout simplement ceux du document JSON.

**Comment by blavenie** [comment]
A noter aussi (tu l'as peut-etre déjà vu), que les graphiques cliquables qui ouvrent cette page permettent de visualiser la syntaxte des requetes envoyées à ES, via le bouton "afficher la requete" (mode expert uniquement) : 

![image](https://user-images.githubusercontent.com/5269556/31791407-c4fe8d02-b518-11e7-8b95-0dcf25d8a14a.png)
